src/Routes.tsx

```diff
@@ -1,84 +1,33 @@
+            workspaceName: pathList[1] || 'default',
+            page: pathList[2] || ''
+        })
+    )
+    if (workspaceName !== workspaceInfo.workspaceName) throw Error('Unexpected mismatch in workspaceName')
 
-const Routes: FunctionComponent<{width: number, height: number}> = ({width, height}) => {
-    return (
-        <Switch>
-            <Route path="/about"><About /></Route>
-            <Route path="/docs"><Docs /></Route>
-            <Route path="/test"><TestPage /></Route>
-            <Route
-                path="/:workspaceName/config"
-                render={({ match }) => (
-                    <Config />
-                )}
-            />
-            <Route
-                path="/:workspaceName/hitherJobMonitor"
-                render={({ match }) => (
-                    <HitherJobMonitor />
-                )}
-            />
-            <Route
-                path="/:workspaceName/importRecordings"
-                render={({ match }) => (
-                    <ImportRecordings />
-                )}
-            />
-            <Route
-                path="/:workspaceName/importSortingsForRecording/:recordingId*"
-                render={({ match }) => (
-                    <ImportSortings recordingId={match.params.recordingId} />
-                )}
-            />
-            <Route
-                path="/:workspaceName/recording/:recordingId*"
-                render={({ match }) => (
-                    <RecordingView recordingId={match.params.recordingId} />
-                )}
-            />
-            <Route
-                path="/:workspaceName/sorting/:sortingId*"
-                render={({ match }) => (
-                    <SortingView sortingId={match.params.sortingId} width={width} height={height} />
-                )}
-            />
-            <Route
-                path="/:workspaceName/sortingUnit/:sortingId/:unitId"
-                render={({ match }) => (
-                    <SortingUnitView sortingId={match.params.sortingId} unitId={parseInt(match.params.unitId)} />
-                )}
-            />
-            <Route
-                path="/:workspaceName/timeseriesForRecording/:recordingId*"
-                render={({ match }) => (
-                    <TimeseriesForRecordingView recordingId={match.params.recordingId} />
-                )}
-            />
-            <Route
-                path="/:workspaceName"
-                render={({ match }) => (
-                    <Home width={width} height={height} />
-                )}
-            />
-            <Route path="/"><Redirect to="/default" /></Route>
-        </Switch>
-    );
+    switch(page) {
+        case 'about': return <About />
+        case 'docs': return <Docs />
+        case 'config': return <Config workspaceInfo={workspaceInfo} />
+        case 'hitherJobMonitor': return <HitherJobMonitor />
+        default: return <Home width={width} height={height} workspaceInfo={workspaceInfo} />
```

Comment:
`useLocation()` is used again within the `Home` component to take care of more fine-grained. It is important to do it that way so that `<WorkspaceView />` can operate as a standalone widget (e.g., for jupyter)